main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/jtblin/gostatsd/statsd"
+
+	"runtime/pprof"
+
+	"syscall"
```

Comment:
These 2 imports should have been added to the other stdlib imports e.g. fmt, etc.

The idiomatic way is to separate imports as:

- standard lib
- internal packages
- external packages